modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -326,9 +325,8 @@ export default {
     },
     async loadNewInstance () {
       this.docReady = false;
-
       const newInstance = await this.getNewInstance();
-
+      this.original = newInstance;
```

Comment:
This could stay as `null` for the sake of quicker evaluation. I might suggest that to save an unnecessary cycle or two.

Reply:
This allows us to determine whether actual effort has been put in to change it from the defaults or not. If we set it null then it is considered never to have changed significantly and the user loses work they may have put in.